firestore/scripts/FriendlyEats.Mock.js

```diff
@@ -59,9 +59,9 @@ FriendlyEats.prototype.addMockRestaurants = function() {
  * Adds a set of mock Ratings to the given Restaurant.
  */
 FriendlyEats.prototype.addMockRatings = function(restaurantID) {
-  const ratingPromises = [];
-  for (let r = 0; r < 5*Math.random(); r++) {
-    const rating = this.data.ratings[
+  var ratingPromises = [];
+  for (var r = 0; r < 5*Math.random(); r++) {
```

Comment:
I just noticed how amazing this is.  The `r < 5 * Math.random()` part will execute on each loop so it's super, duper random.

Reply:
That's pretty funny. I wonder what the distribution of actual `r`'s is lol.